app/constants/view.ts

```diff
@@ -23,6 +23,8 @@ export const HEADER_SEARCH_HEIGHT = SEARCH_INPUT_HEIGHT + 5;
 export const HEADER_SEARCH_BOTTOM_MARGIN = 10;
 
 export const INDICATOR_BAR_HEIGHT = 38;
+export const JOIN_CALL_BAR_HEIGHT = 38;
```

Comment:
Currently it looks like the `INDICATOR_BAR_HEIGHT` is not being used in code yet, so I'd be worried that someone would start using it and change it without realizing we're be depending on it. 
Plus, there's a good chance this will change with the pending UI work. So maybe wait until then and see if it's still the same size? (Even then I'd still worry about the first point, though.)

Reply:
ohh this is dead code I mean everything related to `INDICATOR_BAR_HEIGHT`